MAUI/DataGrid/filtering.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title: Filtering in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Filtering support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Filtering in MAUI DataGrid (SfDataGrid)
+
+Filtering is the process of retrieving the values from the collection which satisfy the specified condition. The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) provides the view filtering support.
+
+## View filtering
+
+The `SfDataGrid` supports filtering the records in the view by setting [SfDataGrid.View.Filter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_Filter) property where `Filter` is a predicate.

```

Comment:
The `SfDataGrid` supports filtering the records in the view by setting the [SfDataGrid.View.Filter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ICollectionViewAdv.html#Syncfusion_Maui_Data_ICollectionViewAdv_Filter) property where the `Filter` is a predicate.

Reply:
Modified,